tests/integration/pidstore/minters/test_minters_arxiv.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2018 CERN.
```

Comment:
2019 :P

Reply:
👍 I'll add a commit which cleans up the licenses